package/samplers/moead/README.md

```diff
@@ -0,0 +1,74 @@
+---
+author: Hiroaki Natsume
+title: MOEA/D sampler
+description: Sampler using MOEA/D algorithm. MOEA/D stands for "Multi-Objective Evolutionary Algorithm based on Decomposition.
+tags: [sampler, multiobjective]
+optuna_versions: [4.0.0]
+license: MIT License
+---
+
+## Abstract
+
+Sampler using MOEA/D algorithm. MOEA/D stands for "Multi-Objective Evolutionary Algorithm based on Decomposition.
+
+This sampler is specialized for multiobjective optimization. The objective function is internally decomposed into multiple single-objective subproblems to perform optimization.
+
+## Class or Function Names
+
+- MOEADSampler
+
+## Installation
+
+```
+pip install scipy
+```
+
+## Example
+
+```python
+import optuna
+import optunahub
+
+def objective(trial: optuna.Trial) -> tuple[float, float]:
+    x = trial.suggest_float("x", 0, 5)
+    y = trial.suggest_float("y", 0, 3)
+
+    v0 = 4 * x**2 + 4 * y**2
+    v1 = (x - 5) ** 2 + (y - 5) ** 2
+    return v0, v1
+
+
+if __name__ == "__main__":
+    population_size = 100
+    n_trials = 1000
+
+    mod = optunahub.load_module("samplers/moead")
+    sampler = mod.MOEADSampler(
+        population_size=population_size,
+        scalar_aggregation_func="tchebycheff",
+        n_neighbors=population_size // 10,
+    )
+    study = optuna.create_study(sampler=sampler)
+    study.optimize(objective, n_trials=n_trials)
```

Comment:
Not a strong opinion, but removing `if __name__ == "__main__":` might be simple as an example.

```suggestion
population_size = 100
n_trials = 1000

mod = optunahub.load_module("samplers/moead")
sampler = mod.MOEADSampler(
    population_size=population_size,
    scalar_aggregation_func="tchebycheff",
    n_neighbors=population_size // 10,
)
study = optuna.create_study(sampler=sampler)
study.optimize(objective, n_trials=n_trials)
```